docs/download.md

```diff
@@ -63,37 +65,40 @@ for your hardware's minimum driver version.
 #### RAPIDS Accelerator's Support Policy for Apache Spark
 The RAPIDS Accelerator maintains support for Apache Spark versions available for download from [Apache Spark](https://spark.apache.org/downloads.html)
 
-### Download v23.10.0
+### Download v23.12.0
+* Download the [RAPIDS
+  Accelerator for Apache Spark 23.12.0 jar - Scala 2.12](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/23.12.0/rapids-4-spark_2.12-23.12.0.jar)
 * Download the [RAPIDS
-  Accelerator for Apache Spark 23.10.0 jar](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/23.10.0/rapids-4-spark_2.12-23.10.0.jar)
+  Accelerator for Apache Spark 23.12.0 jar - Scala 2.13](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.13/23.12.0/rapids-4-spark_2.13-23.12.0.jar)
 
 This package is built against CUDA 11.8. It is tested on V100, T4, A10, A100, L4 and H100 GPUs with 
 CUDA 11.8 through CUDA 12.0.
 
 ### Verify signature
-* Download the [RAPIDS Accelerator for Apache Spark 23.10.0 jar](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/23.10.0/rapids-4-spark_2.12-23.10.0.jar)
-  and [RAPIDS Accelerator for Apache Spark 23.10.0 jars.asc](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/23.10.0/rapids-4-spark_2.12-23.10.0.jar.asc)
+* Download the [RAPIDS Accelerator for Apache Spark 23.12.0 jar](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/23.12.0/rapids-4-spark_2.12-23.12.0.jar)
+  and [RAPIDS Accelerator for Apache Spark 23.12.0 jars.asc](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/23.12.0/rapids-4-spark_2.12-23.12.0.jar.asc)
```

Comment:
The link descriptions should be updated to make it clear these links are for Scala 2.12 (i.e.: be consistent with what was done just above).  There's no link for Scala 2.13 signature file, so it's difficult for users to follow the somewhat vague instruction "use similar steps" without resorting to URL hacking.

IMO we should put the signature links in the download section next to the jar links, and this section should be updated to talk in general terms of how to verify a signature given a set of .jar and .jar.asc files.

Reply:
Sure, thanks for the feedback!